### ISSUE TITLE
Landscape Navigation Bar Too Small on iPad

### ISSUE THREAD

**Issue by SteffeyDev** [react-navigation/react-navigation#2795]
### Current Behavior
- When an iPad is turned to landscape, the nav bar becomes shorter (due to #2669).  This makes the status bar items (such as the time) touch and overlap with the navigation bar items (like the title).

### Expected Behavior
- When an iOS device has a size class of Vertical: Compact, the nav bar should be shortened, because the OS hides the status bar automatically.  With a size class of Vertical: Regular (such as on an iPad in landscape), the nav bar should be full height to accommodate the status bar.

The issue is in this file: https://github.com/react-community/react-navigation/blob/master/src/views/Header/Header.js

### Your Environment

| software         | version
| ---------------- | -------
| react-navigation | 1.0.0-beta.15
| react-native     | 0.48.4
| node             |  7.7.1
| yarn      | 1.1.0

**Comment by vonovak** [comment]
are you sure #2676 is related? It handles tab bar, not navbar.

**Comment by SteffeyDev** [comment]
Sorry, meant #2669. I updated

**Comment by kelset** [comment]
Related to #2794 I think.

**Comment by SteffeyDev** [comment]
@kelset I made both issues and they are related, but still different issues that could be solved independently or at the same time

**Comment by kelset** [comment]
Hey @SteffeyDev , we released beta.17; does your issue still happen in the latest version?

**Comment by SteffeyDev** [comment]
Looks like this was fixed, thanks guys!  I’ll be ready to start tracking the latest version again as soon as #2794 is fixed

**Comment by SteffeyDev** [comment]
@kelset Not sure if I did check completely enough, or if a later version broke this again, but in beta 21 there is still a slight issue.  When my iPad is in landscape, the nav bar is a few pixels shorter, meaning my bar button icons are very cramped.  I can watch the bar change height when I rotate my iPad.  I don't think this behavior is correct, and the nav bar height should always be 44 points plus status bar height on iPad, regardless of orientation.  I verified that Apple's apps behave this way as well.
